modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/jdbc/JdbcUtils.java

```diff
@@ -50,7 +50,7 @@ public static void writeItems(BinaryWriterExImpl writer, List<List<Object>> item
      * @param reader Binary reader.
      * @return Query results items.
      */
-    public static List<List<Object>> readItems(BinaryReaderExImpl reader) {
+    public static List<List<Object>> readItems(BinaryReaderExImpl reader, JdbcProtocolContext ctx) {
```

Comment:
same as above

Reply:
You right. 
This change may be applied when it is needed.
Reverted.